### PR TITLE
Symfony 5 support

### LINKED ISSUE #140: Not setting the attribute "package" of the node "deprecated" in loader.xml
Getting deprecation warnings after a recent update of Symfony.  Seems `package` and `version` attributes for the `<deprecated/>` tag are now required.

### PR DESCRIPTION
**Pull request by chalasr**
Replaces #125, #139 
Closes #140

### CODE REVIEW

**Comment by StefanRo98** [comment]
When will this be ready to merge?

**Comment by theofidry** [comment]
@StefanRo98 when someone will be willing and manage to fix the build https://travis-ci.org/github/theofidry/AliceDataFixtures/builds/654897348

**Comment by jmsche** [comment]
Congrats @chalasr :) :tada:

File: fixtures/Bridge/Symfony/SymfonyApp/config/config_doctrine_phpcr.yml [hunk 1]
```diff
@@ -18,6 +18,9 @@ doctrine:
                 password: ~
                 host: 127.0.0.1
                 port: 3307
+                charset: UTF8
+                options:
+                    1002: "SET NAMES 'UTF8' COLLATE 'utf8_unicode_ci'"
```
Comment (chalasr) [inline fixtures/Bridge/Symfony/SymfonyApp/config/config_doctrine_phpcr.yml:23]: required to workaround a bug related to the doctrine-phpcr integration (https://github.com/doctrine/DoctrinePHPCRBundle/issues/310) and keep tests green on lowest/highest supported versions

**Comment by chalasr** [comment]
@jmsche Thanks!
@theofidry Green it is 🎉

**Comment by jmsche** [comment]
@chalasr I guess I can close https://github.com/theofidry/AliceDataFixtures/pull/139? :)

File: fixtures/Bridge/Symfony/SymfonyApp/config/config_doctrine_phpcr.yml (hunk 1 above)
Comment (theofidry) [reply fixtures/Bridge/Symfony/SymfonyApp/config/config_doctrine_phpcr.yml:24]: Could you add that link to the command?

Comment (chalasr) [reply fixtures/Bridge/Symfony/SymfonyApp/config/config_doctrine_phpcr.yml:24]: Done

Comment (chalasr) [reply fixtures/Bridge/Symfony/SymfonyApp/config/config_doctrine_phpcr.yml:24]: Done

**Comment by chalasr** [comment]
@jmsche Yes, sorry about that, I needed to fix this case to make travis green again, and the approach is a bit more complex because tests are run against lower symfony versions.

**Comment by jmsche** [comment]
@chalasr Not a problem at all :)

**Comment by theofidry** [comment]
Thank you a lot @chalasr for the work!

**Comment by StefanRo98** [comment]
@chalasr Thank you!

**Pull request merged**